### PR TITLE
feat: bump the cosmos-sdk version to `v0.47.1`

### PR DESCRIPTION
**Pull request by Pantani**
## Description

closes: #XXXX

## Tasks

- [ ] 

## References

- 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/docs/building-modules/README.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Primary repo maintainers
+*       @ilgooz @jeronimoalbi @Pantani
```
Comment (jeronimoalbi) [inline .github/CODEOWNERS:4]: GitHub is warning that `@Pantani` is an unknown owner. Maybe a permissions issue?

**Review by jeronimoalbi — APPROVED** [review]
Two possible minor issues I noticed is that the default make target fails, but that happens already in the main branch, and that the app won't run with Ignite CLI by default, it would require an automatic config upgrade to v2 and `ignite doctor` to add tools dependencies.
In any case I believe we could make these corrections in a different PR if required.

File: .github/CODEOWNERS (hunk 1 above)
Comment (Pantani) [reply .github/CODEOWNERS:4]: yes, weird, maybe because I opened the PR from a fork. I think it will solve after the merge

**Comment by Pantani** [comment]
> Two possible minor issues I noticed is that the default make target fails, but that happens already in the main branch, and that the app won't run with Ignite CLI by default, it would require an automatic config upgrade to v2 and `ignite doctor` to add tools dependencies. In any case I believe we could make these corrections in a different PR if required.

nicee!! thanks for catching it. let's solve this in another PR. Can you open these 2 issues, please?

**Pull request merged**